scripts/bootstrap/bootstrap-haskell.ps1

```diff
@@ -653,10 +655,15 @@ if (!($DontAdjustBashRc)) {
   $AdjustBashRcExport = 'export BOOTSTRAP_HASKELL_ADJUST_BASHRC=1 ;'
 }
 
+if (!($Msys2Env)) {

```

Comment:
What's the logic here? Looks to me this is testing whether `Msys2Env` is _not_ set and, if so, sets `Msys2EnvExport` and `GHCUP_MSYS2_ENV` based on it?

Reply:
Yes, this is not finished/correct.

We want to set a default, when `Msys2Env` is not set.